tests/Block/FormatterBlockServiceTest.php

```diff
@@ -37,7 +38,7 @@ public function testExecute(): void
         $this->assertSame('@SonataFormatter/Block/block_formatter.html.twig', $this->templating->view);
 
         $this->assertInternalType('array', $this->templating->parameters['settings']);
-        $this->assertInstanceOf('Sonata\BlockBundle\Model\BlockInterface', $this->templating->parameters['block']);
+        $this->assertInstanceOf(BlockInterface::class, $this->templating->parameters['block']);
```

Comment:
IMO this kind of changes should be done on 3.x instead, like we did for the other bundles

Reply:
Well, the "plan" was to do a major release because of license issue, if we do it, there is no point doing it on 3.x then.

Reply:
Not sure how soon this next major release will be.

Reply:
I would rather close this than revert half of the changes 😅

Reply:
`git reset --patch` should make it easy. I'll do it for you.

Reply:
did not know about this, ty

Reply:
Wasn't that easy, you made so many changes :sweat_smile: , would have been easier to reset to origin/master and then `git add --patch`